avm/src/vm/opcode.rs

```diff
@@ -3563,6 +3563,16 @@ pub static OPCODES: Lazy<HashMap<i64, ByteOpcode>> = Lazy::new(|| {
     }
     None
   });
+  cpu!(getM => fn(args, hand_mem) {
```

Comment:
why not `getMaybe`? Outside the context of this PR, it would be hard to grasp the difference between `getM` and `getMain` or what the `M` in `getM` stands for